apps/api/pages/api/bookings/_get.ts

```diff
@@ -208,18 +208,13 @@ async function handler(req: NextApiRequest) {
     args.where = buildWhereClause(userId, attendeeEmails, [], []);
   }
 
-  if (dateFrom && dateTo) {
-    args.where = {
-      ...args.where,
-      startTime: { gte: dateFrom },
-      endTime: { lte: dateTo },
-    };
-  } else if (dateFrom) {
+  if (dateFrom) {
     args.where = {
       ...args.where,
       startTime: { gte: dateFrom },
     };
-  } else if (dateTo) {
+  }
+  if (dateTo) {
     args.where = {
       ...args.where,
       endTime: { lte: dateTo },
```

Comment:
@alishaz-polymath I was able to simplify a bit but I can't directly set `args.where.startTime` or `args.where.endTime` because `args.where` is possibly undefined - so we need to create `args.where` if it doesn't exit. This feels pretty clean to me but LMK!